substrate/utils/frame/benchmarking-cli/src/pallet/types.rs

```diff
@@ -32,8 +32,10 @@ pub enum GenesisBuilder {
 	None,
 	/// Let the runtime build the genesis state through its `BuildGenesisConfig` runtime API.
 	Runtime,
+	// Use the runtime from the Spec file to build the genesis state.
+	SpecRuntime,
 	/// Use the spec file to build the genesis state. This fails when there is no spec.
-	Spec,
+	SpecGenesis,
```

Comment:
nit: Maybe better name for this while enum would be `GenesisBuilderPolicy`?

Reply:
Yes sounds better, the original name was a bit random 😅